crypto/keys/hd/hdpath_test.go

```diff
@@ -3,6 +3,7 @@ package hd
 import (
 	"encoding/hex"
 	"fmt"
+	"github.com/stretchr/testify/require"
```

Comment:
IntelliJ idea strikes again! :) It'd be good to sort the imports nicely

Reply:
just a matter of setting things up ;)
Automatically using goimports now